bc_obps/service/data_access_service/operator_service.py

```diff
@@ -0,0 +1,23 @@
+from registration.models import Operator
```

Comment:
Bri figure out how to do the model thing Pierre was talking about

Reply:
@pbastia , I didn't understand what you meant by this comment in the other PR: `Maybe if we want to namespace a little better like DataAccessService.Operators.create(...):), this would be easy with python's folder/module structure`. Have I got it set up like you were imagining?